manipulation/src/main/java/org/jboss/gm/manipulation/actions/PomTransformer.java

```diff
@@ -0,0 +1,91 @@
+package org.jboss.gm.manipulation.actions;
+
+import static org.jboss.gm.common.ProjectVersionFactory.withGAV;
+
+import org.commonjava.maven.atlas.ident.ref.ProjectVersionRef;
+import org.gradle.api.Action;
+import org.gradle.api.XmlProvider;
+import org.jboss.gm.common.alignment.AlignmentModel;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+/**
+ * Overrides data in pom generated by the old 'maven' plugin.
```

Comment:
This comment is in fact wrong, because this transformer is used also for the maven-publish plugin.